README.md

```diff
@@ -8,3 +8,68 @@ Branches:
 * `XS-x.y` (e.g. `XS-7.5`), when they exist, contain sources from the `x.y` release of XenServer, with trademarked or copyrighted material stripped if needed.
 
 Built RPMs and source RPMs are available on https://updates.xcp-ng.org.
+
+---
+
+## :crystal_ball: Information & Remarks
+
+This Linux kernel module adds support to XCP-ng for NICs based on Marvell _(originally Aquantia)_ AQC111U over USB3. The driver should not be confused with the generic AQC111 that supports the whole family of NICs based on the AQC111 chipset, but NOT the ones connected over USB3. This kernel module provides support only for AQC111U-based NICs.
+
+The kernel module has been tested in a homelab environment with both XCP-ng version `8.1` and `8.2`.
+
+Known compatible NICs are<sup id="nt1">[1](#fnt1)</sup>:
+
+| Brand    | Model      | Tested             |
+|----------|:----------:|:------------------:|
+| QNAP     | QNA-UC5G1T | :white_check_mark: |
+| Sabrent  | NT-SS5G    | :white_check_mark: |
+| StarTech | US5GA30    |                    |
+| StarTech | US5GC30    |                    |
+| Trendnet | TUC-ET5G   |                    |
+
+
+## :hammer: How to build
+
+In order to build the RPM package a machine running Linux and providing Docker is necessary. To create the package follow the steps below:
+
+```bash
+$ mkdir -p aqc111u-module-build
+$ cd aqc111u-module-build
+$ git clone https://github.com/xcp-ng/xcp-ng-build-env.git
+$ git clone https://github.com/xcp-ng-rpms/aqc111u-module.git
+$ ls xcp-ng-build-env/Dockerfile* | \
+      xargs -I %%%% sed -i \
+      's~RUN *useradd builder~RUN groupadd -g '"$(id -g)"' builder \&\& useradd -u '"$(id -u)"' -g '"$(id -g)"' builder~g' \
+      %%%%
+$ ./xcp-ng-build-env/run.py -b <X.Y> --build-local aqc111u-module/ --rm
+
+```
+
+The value `<X.Y>` should be replaced with the version of XCP-ng the driver is planned to be used for _(e.g. 7.5, 8.0, 8.2, etc.)_. At the end of the build process a RPM package will be avaialble inside the `aqc111u-module/RPMS/x86_64/` directory.
+
+
+## :test_tube: How to use
```

Comment:
How do you prefer me to address this?

- the changes should be removed and the `https://xcp-ng.org/docs/` site should have the modifications added? _(How?)_
- leave the info here and just split it as suggested?

Thanks

Reply:
I would prefer to have it in the docs. Each page there has a "Help us improve this page" link that allows to make changes. Or you can directly create a PR against https://github.com/xcp-ng/xcp-ng-org/

Reply:
Commit 0f08735 removes the section dedicated to the AQC111U information in favor of a the [dedicated PR #97 in xcp-ng/xcp-ng-org documetation](https://github.com/xcp-ng/xcp-ng-org/pull/97)

Commit d6b61b2 removes _(without any replacement elsewhere)_ the section dedicated to the use of `udev` rules to avoid pollution of unused PIFs upon reboot _(or pugging/unplugging)_ that applies, in general, to all USB NICs in XCP-ng. Not really sure where to put that content, but here it is clearly not the right place...